### PR TITLE
Streamline Docker-Compose for all services

### PR DESCRIPTION
**Pull request by dkarlovi**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| License       | MIT

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ services:
         volumes:
             - pg-data:/var/lib/postgresql/data:rw
         ports:
-            - "5432:55432"
+            - "55432:5432"
```
Comment (dkarlovi) [inline docker-compose.yaml]: @sstok it's HOST:CONTAINER, not CONTAINER:HOST :laughing:

Comment (sstok) [reply docker-compose.yaml]: Man I feel stupid now 🤦‍♂️ 

Why, why must life be so hard

**Comment by sstok** [comment]
Thank you @dkarlovi

File: docker-compose.yaml (hunk 1 above)
Comment (dkarlovi) [reply docker-compose.yaml]: FWIW, CONTAINER:HOST makes sense to me too, HOST:CONTAINER feels backwards.

**Pull request merged**